lib/ssh_scan/database/sqlite_db.rb

```diff
@@ -0,0 +1,65 @@
+require 'sqlite3'
+
+module   SSHScan
+  module Database
+    class SQLite
+      def initialize(database_name = "api.db")
+        if File.exist?(database_name)
+          @db = ::SQLite3::Database.open(database_name)
+        else
+          @db = ::SQLite3::Database.new(database_name)
+          create_schema
+        end
+      end
+
+      def create_schema
+        @db.execute <<-SQL
+          create table api_schema (
+            worker_id varchar(100),
+            uuid varchar(100),
+            result json,
+            scanned_on datetime
+          );
+        SQL
+      end
+
+      def add_scan(worker_id, uuid, result)
+        @db.execute "insert into api_schema values ( ? , ? , ? , ? )",
+                    [worker_id, uuid, result.to_json, Time.now.to_s]
+      end
+
+      def delete_scan(worker_id, uuid)
+        @db.execute(
+          "delete from api_schema where worker_id = ? and uuid = ?",
+          worker_id, uuid
+        )
+      end
```

Comment:
Should you need to know the worker_id to delete a scan?  Functionally, I would think the uuid would be enough.